### PR TITLE
MD-4815: Updated baselines due to changes in DRILL-6492

### PR DESCRIPTION
**Pull request by agozhiy**
Need to be merged after this Drill PR:
https://github.com/apache/drill/pull/1439

### CODE REVIEW

File: framework/resources/Functional/misc/describe1.e_tsv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-dfs_describe	\{
+dfs_describe.default	\{
```
Comment (Agirish) [inline framework/resources/Functional/misc/describe1.e_tsv]: Not clear why we need this.

File: framework/resources/Functional/misc/describe1.e_tsv [hunk 2]
```diff
@@ -10,7 +10,7 @@ dfs_describe	\{
       \"escape\" : \"`\",
       \"comment\" : \"\^\",
       \"skipFirstLine\" : true,
-      \"delimiter\" : \"|\"
+      \"delimiter\" : \"\|\"
```
Comment (Agirish) [inline framework/resources/Functional/misc/describe1.e_tsv]: Not clear why we need this, either.

**Comment by arina-ielchiieva** [comment]
@agozhiy please reply for Abhishek comments.
@agozhiy / @Agirish please merge the changes asap since DRILL-6492 was merged into Apache master.

File: framework/resources/Functional/misc/describe1.e_tsv (hunk 1 above)
Comment (agozhiy) [reply framework/resources/Functional/misc/describe1.e_tsv]: This is one of the changes made by Arina:
> Default workspace name was added to the output. Example: hive -> hive.default

Comment (arina-ielchiieva) [reply framework/resources/Functional/misc/describe1.e_tsv]: Yes, we new changes when we describe schema, we show full path, including workspace.

File: framework/resources/Functional/misc/describe1.e_tsv (hunk 2 above)
Comment (agozhiy) [reply framework/resources/Functional/misc/describe1.e_tsv]: '|' is an alternation symbol in regex, so it need to be escaped.

**Pull request merged**